client/lib/oauth2-clients.js

```diff
@@ -6,6 +6,10 @@
 
 import { includes } from 'lodash';
 
+export const isCrowdsignalOAuth2Client = oauth2Client => {
+	return oauth2Client.id === 978;
```

Comment:
Assume this is cs oauth client? Maybe extract constant.

Reply:
It is. I'm going to leave it as is for just right now as we don't really need it to do anything else right now.  
It's based on a similar function for Woo underneath.